fpdf/fpdf.py

```diff
@@ -34,8 +34,14 @@
 from typing import Callable, List, NamedTuple, Optional, Tuple, Union
 from fontTools import ttLib
 from fontTools import subset as ftsubset
+from fontTools import version as ftversion
 from io import BytesIO
 
+if ftversion != "4.37.1":
```

Comment:
I do not think this warning is necessary, given the change made to `setup.py`

Reply:
Given your comment on the pinned dependency below, I agree with you.